aiven/client/cli.py

```diff
@@ -5666,6 +5666,64 @@ def service__flink__cancel_application_deployments(self) -> None:
             ),
         )
 
+    @arg.project
+    @arg.service_name
+    def service__custom_file__list(self) -> None:
+        """List all the custom files for the specified service"""
+        self.print_response(
+            self.client.custom_file_list(
+                project=self.args.project,
+                service=self.args.service_name,
+            ),
+        )
+
+    @arg.project
+    @arg.service_name
+    @arg("--file_id", help="A file ID, usually an uuid4 string", required=True)
+    def service__custom_file__get(self) -> None:
+        """View the custom file for the specified service and ID"""
+        print(
```

Comment:
Looking at the upload implementation in the client, the custom files are `application/octet-stream` i.e. binary data, does `print()` always write to stdout such data exactly as it, bit-by-bit?

And with binary data, an option to output to a file could be a convenient option (though one can do a shell redirect too of course).

Reply:
Since we assume cutom files to be utf-8 encoded text files, it should be save to simply redirect the output of `print`

Reply:
Custom files going forward can be anything. I would not have any implementation that prints to stdout but require target file name and write there. Some special syntax to allow using stdout as the target file could be allowed.